docs/api-gateway-v2/03-04-migration.md

```diff
@@ -44,105 +44,22 @@ Follow these steps:
     kubectl get api {NAME} -n {NAMESPACE} -o yaml
     ```
 
-2. Create an APIRule CR based on the Api CR's specification:
-
->**NOTE:** This step focuses on the`spec` part of the CRD. You can `metadata` field in a preferred way and do not copy the [`status`](components/api-gateway-v2#custom-resource-api-rule-additional-information) field from the original Api CR.
-
-a. Replace the default value (`kyma-gateway.kyma-system.svc.cluster.local`) for the **gateway** parameter with Istio Gateway used to expose services. 
-
-b. Copy the values for **service.name** and **service.port** parameters from the corresponding fields of the Api CR. 
-
-c. Set the value for **service.host** parameter to any temporary value which includes the domain. For example, if the value for the **hostname** parameter of the Api CR was set to `sample-service.kyma.local`, change it to `temp-sample-service.kyma.local` so that differs from the original value. Make sure other services on your cluster do not use this hostname. 
-
-d. Configure the **rules** parameter of APIRule CR.
-
->**NOTE:** For details on authentication configuration and possible values you can use, see [this](#authentication-of-api-resources-and-apirules) section.
-
-When the configuration is ready, create the APIRule object and test if the service is working as expected on the new host. It should work the same on both hosts.
-
-
-3. Remove the dependent resources of the Api CR:
-
-   ```shell script
-     kubectl delete virtualservice {API_NAME} -n {NAMESPACE}
-    ```
-Make sure that the Virtual Service resource is deleted before proceeding.
-
-4. If the Api CR was secured with an authentication mechanism, delete the Policy resource:
-
-    ```shell script
-     kubectl delete policy {API_NAME} -n {NAMESPACE}
-    ```
-
-5. To make the service returns to its original host, set the **spec.service.host** parameter of APIRule CR to the value used by the Api CR.
-
-    ```shell script
-    kubectl edit apirule {APIRULE_NAME} -n {NAMESPACE}
-    ```
-
-After saving that configuration, the service should be available on the original hostname.
-
-6. Remove the Api CR:
-
-    ```shell script
-    kubectl delete api {API_NAME} -n {NAMESPACE}
-    ```
-
-## Authentication of Apis and APIRules
+2. Create an APIRule CR based on the Api CR's specification.
 
-To properly configure the authentication mechanism for the new APIRule, learn more about the differences in authentication configuration for Api and APIRule CRs:
+>**NOTE:** Do not copy the [**status**](components/api-gateway-v2#custom-resource-api-rule-additional-information) parameter from the original Api CR.
 
-| Api CR       | APIRule CR| 
-|--------------| ----------|
-|Configured with a list of JWT issuers set for all paths of a service.| Configured with a list of path definitions along with their respective authentication methods.|
```

Comment:
IMHO the information in this table is important - it explains the most important difference between both Custom Resources.

Reply:
It's there, just at a later point :) After both examples.